ArcadiaTeamsBot.CQRS.Abstractions/Queries/GetCurrentServiceDeskRequestsQuery.cs

```diff
@@ -1,9 +1,11 @@
 ﻿namespace ArcadiaTeamsBot.CQRS.Abstractions
 {
     using MediatR;
-    using ServiceDesk.Abstractions.DTOs;
```

Comment:
You don't have to do this manually, you should correctly configure your R#. System usings should be placed prior to all others.